internal/cli/opsmanager/admin/backup/blockstore/delete.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package blockstore
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type DeleteOpts struct {
+	*cli.DeleteOpts
+	store store.BlockstoresDeleter
+}
+
+func (opts *DeleteOpts) init() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *DeleteOpts) Run() error {
+	return opts.Delete(opts.store.DeleteBlockstore)
```

Comment:
[q] does this work? doesn't this method assume a project ID? which you may not have?

Reply:
This endpoint does not need the projectID https://docs.opsmanager.mongodb.com/current/reference/api/admin/backup/snapshot/mongoConfigs/delete-one-blockstore-configuration/.

Reply:
correct, so my question is will `opts.Delete` work, as I thought it handled the project ID, unless it doesn't

Reply:
Yes, I tested it and It works. I don't think the ProjectID is handled by opts.Delete (I quick looked at the code)